lang/src/accounts/program.rs

```diff
@@ -22,9 +22,15 @@ use std::ops::Deref;
 /// - `Account.info.key == Program`
 /// - `Account.info.executable == true`
```

Comment:
could you adjust these checks as well while youre here?

`account_info` instead of `Account.info`.